docs/morevp.md

```diff
@@ -351,18 +387,32 @@ It’s possible for an honest user to start an exit and have their exit bond sla
 This can occur if one of the inputs to a transaction is malicious and signs a second transaction spending the same input.
 
 The following scenario demonstrates this attack:
-1. Mallory spends `UTXO1` in `TX1` to Bob, creating `UTXO2`.
+1. Mallory spends `UTXO1m` and Alice spends `UTXO1a` in `TX1` to Bob, creating `UTXO2`.
 2. `TX1` is in-flight.
 3. Operator begins withholding blocks while `TX1` is still in-flight.
-4. Bob starts an exit referencing `TX1` and places `exit bond`.
-5. Mallory spends `UTXO1` in `TX2`.
+4. Alice starts an exit referencing `TX1` and places `exit bond`.
+5. Mallory spends `UTXO1m` in `TX2`.
 6. In period 1 of the exit for `TX1`, Mallory challenges the canonicity of `TX1` by revealing `TX2`.
 7. No one is able to respond to the challenge in period 2, so `TX1` is determined to be non-canonical.
-8. After period 2, Mallory receives Bob’s `exit bond`, no one exits any UTXOs.
+8. After period 2, Mallory receives Alice's `exit bond`, no one exits any UTXOs.
 
-Mallory has therefore caused Bob to lose `exit bond`, even though Bob was acting honestly.
+Mallory has therefore caused Alice to lose `exit bond`, even though Alice was acting honestly.
 We want to mitigate the impact of this attack as much as possible so that this does not prevent users from receiving funds.
 
+#### Honest transaction retries attack
+
+Retrying a transaction that has failed for a trivial reason is not safe under MoreVP.
+
+Scenario is:
+1. Honest Alice creates/signs/submits a transaction `tx1`
+2. This fails, either tacitly or loudly - `tx1` doesn't get included in a block
```

Comment:
`either tacitly or loudly` - it might be just me but don't get it - can you rephrase please.

Reply:
it means either an error response will be returned from the child chain server or just silence. In both situations Alice is in trouble. I'll clarify